static/js/hotkey.js

```diff
@@ -701,6 +701,7 @@ exports.process_hotkey = function (e, hotkey) {
     case 'message_actions':
         return popovers.open_message_menu(msg);
     case 'star_message':
+        e.preventDefault();
```

Comment:
Why did you add this here?  This is done in the caller.

Reply:
I get a normal save dialog box without adding this line here.

Reply:
Ahh, the bug is on the next line.  We should be doing `return true` after calling `message_flags.toggle_starred(msg);
`, not returning undefined.